src/main/resources/logback.xml

```diff
@@ -82,16 +82,18 @@
   <!--
     | Setup default log level to INFO
   +-->
-  <root level="INFO">
+  <root level="WARN">
     <appender-ref ref="LOG" />
   </root>
 
   <!--
     | Turn up logging for portlet specific package
    +-->
-  <!--
   <logger name="edu.wisc.my.portlets.bookmarks" additivity="false" level="DEBUG">
     <appender-ref ref="LOG" />
   </logger>
-  -->
+  <logger name="org.springframework.web" additivity="false" level="INFO">
```

Comment:
Could this be removed in favor of `<root level="WARN">` and change the `level` there to `INFO`?

Reply:
No, defaulting to INFO provided way too much Hibernate logging when I really wanted to focus on Spring Web code. I'll level it inside a comment. Thanks!

Reply:
Sounds familiar, I feel like I've seen a logback.xml that quieted down the Hibernate log entries. I'll see if I can find it ...

Reply:
Could https://github.com/uPortal-Project/uPortal/blob/master/uPortal-webapp/src/main/resources/logback.xml#L182-L184 be what you would want?

Reply:
Hmm, I'd rather let the WARN default handle all that without getting into specifics, especially with Hibernate changing so much. Let's leave the logger entries as is for now. That work?

Reply:
I'm good with that :+1: